src/main/java/com/ciandt/techgallery/ofy/OfyService.java

```diff
@@ -0,0 +1,69 @@
+package com.ciandt.techgallery.ofy;
+
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+
+import com.ciandt.techgallery.persistence.model.Card;
+import com.ciandt.techgallery.persistence.model.UserGroup;
+import com.ciandt.techgallery.sample.persistence.model.Recommendation;
+import com.ciandt.techgallery.sample.persistence.model.Sample;
+import com.ciandt.techgallery.sample.persistence.model.Technology;
+import com.googlecode.objectify.Objectify;
+import com.googlecode.objectify.ObjectifyFactory;
+import com.googlecode.objectify.ObjectifyService;
+
+/**
+ * Service class for Objectify settings.
+ * 
+ * @author felipers
+ *
+ */
+public class OfyService implements ServletContextListener {
+
+  /** Define all entities first. */
+  static {
+    // ObjectifyService.register(User.class);
+    ObjectifyService.register(UserGroup.class);
+    ObjectifyService.register(Card.class);
+    ObjectifyService.register(Technology.class);
+    ObjectifyService.register(Recommendation.class);
+    ObjectifyService.register(Sample.class);
```

Comment:
There is a TODO to fix later because these are samples... We must to find another way to separate these examples from the production's entities.